packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -53,23 +57,36 @@ class BlockListBlock extends Component {
 
 	getBlockForType() {
 		return (
-			<BlockEdit
-				name={ this.props.name }
-				isSelected={ this.props.isSelected }
-				attributes={ this.props.attributes }
-				setAttributes={ this.props.onChange }
-				onFocus={ this.onFocus }
-				onReplace={ this.props.onReplace }
-				insertBlocksAfter={ this.insertBlocksAfter }
-				mergeBlocks={ this.props.mergeBlocks }
-				onCaretVerticalPositionChange={
-					this.props.onCaretVerticalPositionChange
-				}
-				clientId={ this.props.clientId }
-				parentWidth={ this.props.parentWidth }
-				contentStyle={ this.props.contentStyle }
-				onDeleteBlock={ this.props.onDeleteBlock }
-			/>
+			<GlobalStylesContext.Consumer>
+				{ ( globalStyle ) => (
+					<GlobalStylesContext.Provider
+						value={ {
+							...globalStyle,
+							...( this.props.wrapperProps || {} ).style,
```

Comment:
Is this safe when `this.props.wrapperProps` is not defined? will `{}.style` work?

Reply:
Yes, it works fine but i think i could remove that fallback since the `wrapperProps` comes from the compose and I set there `{}` as a default value :)
https://github.com/WordPress/gutenberg/pull/21637/files#diff-42b230fc8f23c8ce7322e2d6ff6d5df6R307